main.cpp

```diff
@@ -51,86 +51,86 @@ arm.flex();                             // 3)
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  1)          
+//  Noun:        swimmer
+//  action 1:    the swimmer dives into water
+swimmer.dive();
+//  action 2:    the swimmer does the backstroke
+swimmer.backstroke();
+//  action 3:    the swimmer does breaststroke
+swimmer.breaststroke();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        radio
+//  action 1:    the radio plays the cassette
+radio.playCassette();
+//  action 2:    the radio plays the CD
+radio.playCD();
+//  action 3:    the radio plays the mp3 file
+radio.playMP3();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        traffic light
+//  action 1:    the traffic light turns red
+trafficLight.turnRed();
+//  action 2:    the traffic light turns yellow
+trafficLight.turnYellow();
+//  action 3:    the traffic light turns green
+trafficLight.turnGreen();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        lion
+//  action 1:    the lion hunts 
+lion.hunt();
+//  action 2:    the lion eats
+lion.eats();
+//  action 3:    the lion roars
+lion.roar();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        dog
+//  action 1:    the dog barks
+dog.bark();
+//  action 2:    the dog bites
+dog.bite();
+//  action 3:    the dog sleeps
+dog.sleep();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        quarterback
+//  action 1:    the quarterback throws the ball
+quarterback.throwBall();
+//  action 2:    the quarterback fumbles the ball
+quarterback.fumbleBall();
+//  action 3:    the quarterback runs with ball
+quarterback.runBall();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        pitcher
+//  action 1:    the pitcher throws fastball
+pitcher.throwFastball();
+//  action 2:    the pitcher throws curveball
+pitcher.throwCurveball();
+//  action 3:    the pitcher throws changeup
+pitcher.throwChangeup();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        jukebox 
+//  action 1:    the jukebox accepts money
+jukebox.acceptMoney();
+//  action 2:    the jukebox accepts song selection
+jukebox.acceptSongSelection();
+//  action 3:    the jukebox plays song
+jukebox.playSong();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:        store clerk
+//  action 1:    the store clerk accepts money
+storeClerk.acceptMoney();
+//  action 2:    the store clerk counts change
+storeClerk.countChange();
+//  action 3:    the store clerk gives change to customer
+storeClerk.giveChange();
+//  10)          
+//  Noun:        elevator
+//  action 1:    the elevator goes up
+elevator.goUp();
+//  action 2:    the elevator goes down
+elevtor.goDown();
+//  action 3:    the elevator stops
+elevator.stop();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.